### DISCUSSION TITLE
sql server 2008 老一点的版本有计划之后支持么？很多老项目有这个需求

### DISCUSSION THREAD

**Discussion by JasonFang1993** [apache/flink-cdc#1482] [Ideas]
Currently Flink SqlServer CDC connector only supports SqlServer whose version is larger or equal to 11, but actual is 10 目前报错 不支持

**Comment by 1lo0** [comment]
@JasonFang1993 这个问题解决了吗

**Comment by GOODBOY008** [comment]
> Currently Flink SqlServer CDC connector only supports SqlServer whose version is larger or equal to 11, but actual is 10 目前报错 不支持

你的 sqlserver 是什么版本？ 2008 r2 之后才支持 cdc。
